una.py

```diff
@@ -0,0 +1,59 @@
+from madina import Zonal
+
+
+class UNA:
+    """
+    A set of network tools including betweenness, reach and gravity analysis.
+    """
+    DEFAULT_DETOUR_RATIO = 1.15
+    DEFAULT_SEARCH_RADIUS = 800
+    IS_TURN_PENALTY = False
+
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def parallel_betweenness(zonal: Zonal, search_radius, detour, decay, decay_method, beta, path_detour_penalty):
+        """
+        TODO: fill function spec
+        """
+        raise NotImplementedError
+
+    @staticmethod
+    def best_path_generator(
+            zonal: Zonal,
+            origin_index,
+            search_radius=DEFAULT_SEARCH_RADIUS,
+            detour_ratio=DEFAULT_DETOUR_RATIO,
+            turn_penalty=IS_TURN_PENALTY
+    ):
+        """
+        TODO: Fill out function spec
+        """
+
+        raise NotImplementedError
+
+    @staticmethod
+    def visualize_graph(zonal: Zonal):
```

Comment:
This should probably be moved to the `Zonal` object

Reply:
Agree, or better yet, the network class as it only depends on the network structure..